qadi_multi_agent.py

```
@@ -0,0 +1,215 @@
+    with open(env_path) as f:
+        for line in f:
+            if line.strip() and '=' in line and not line.startswith('#'):
+                key, value = line.strip().split('=', 1)
+                os.environ[key] = value.strip('"').strip("'")
+
+sys.path.insert(0, str(Path(__file__).parent / 'src'))
+
+async def run_full_qadi_system(prompt: str, use_evolution: bool = False):
+    """Run the full QADI multi-agent system with optional genetic evolution."""
+    from mad_spark_alt.core.smart_orchestrator import SmartQADIOrchestrator
+    from mad_spark_alt.core.smart_registry import smart_registry
```

Ruff (0.11.9)

`mad_spark_alt.core.smart_registry.smart_registry` imported but unused

Remove unused import: `mad_spark_alt.core.smart_registry.smart_registry`

(F401)